action-scripts/copy-theme.js

```diff
@@ -8,7 +8,7 @@ const copyThemeFile = async ({ outputFolder, styleSystem, actionPath }) => {
   }[styleSystem];
 
   const themeSource = path.join(actionPath, "theme_templates", themeTemplate);
-  const themeDest = path.join(outputFolder, styleSystem, themeTemplate);
+  const themeDest = path.join(outputFolder, themeTemplate);
```

Comment:
Removed the `styleSystem` so the generated `theme_template` file is output to the same directory as the other generated files.